spec/unit/validationDelete.spec.ts

```diff
@@ -206,20 +206,26 @@ describe('ValidationDynamoDbManager - Delete()', function ()
             assert.isUndefined(error)
         });
 
-        it('valid - hash/range property', async () =>
+        it('invalid - hash/range property', async () =>
         {
-            await manager.delete(Entity, 42, {conditionExpression: 'condition', expressionAttributeNames: {'#n': 'hashProperty', "#m": 'rangeProperty'}});
+            try
+            {
+                await manager.delete(Entity, 42, {conditionExpression: 'condition', expressionAttributeNames: {'#n': 'hashProperty', "#m": 'rangeProperty'}});
+            }catch(e) { error = e; }
             
-            assert.isTrue(called);
-            assert.isUndefined(error);
+            assert.isFalse(called);
+            assert.instanceOf(error, ValidationError);
         });
 
         it('valid - using the real property name instead of a custom name', async () =>
         {
-            await manager.delete(Entity, 42, {conditionExpression: 'condition', expressionAttributeNames: {'#n': 'realProperty'}});
+            try
+            {
+                await manager.delete(Entity, 42, {conditionExpression: 'condition', expressionAttributeNames: {'#n': 'realProperty'}});
+            }catch(e) { error = e; }
 
-            assert.isTrue(called);
-            assert.isUndefined(error);
+            assert.isFalse(called);
```

Comment:
We cannot, because custom names are for hashes and ranges.

Hashes and ranges are converted to 'hash' and 'range' which are different values for each representation.

Reply:
i will adjust name to invalid